app/jobs/regular/invite_email.rb

```diff
@@ -15,8 +15,11 @@ def execute(args)
 
       message = InviteMailer.send_invite(invite)
       Email::Sender.new(message, :invite).send
-    end
 
+      if invite.emailed_status == Invite.emailed_status_types[:sending]
```

Comment:
Is there a good reason to only update the status to `sent` if it was sending? I feel like if the email was sent it could always be updated.

Reply:
The only exception I think where we should not update the status to `sent` is when the status is `not required`. That is because when the invite link is generated we set invite `emailed_status` to 'not required'. Now the inviter has invite link with him. If inviter sends same invite again via email this time the `emailed_status` will be updated to 'sent' and that invite link (generated via invite link feature) can be used to create an active account (as per [this code](https://github.com/discourse/discourse/pull/7875/files#diff-2a355fe6078d7a0ee0e440532e4032e2R63)) without verifying the email which can be classified as security issue.

I will change to code to update status to `sent` if the original status was `pending` or `sending`.